modules/uncertainty/R/generate_joint_ensemble_design.R

```diff
@@ -85,7 +96,5 @@ generate_joint_ensemble_design <- function(settings,
     return(sobol_obj)
   }
 
-
-
   return(list(X = design_matrix))
```

Comment:
Is it important that this be a list? All the uses I've seen so far immediately subset the result to only store the X component, so would it make sense to return `design_matrix` unwrapped in the first place?